waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/usecase/claim/SilentClaimStrategy.kt

```diff
@@ -60,7 +65,9 @@ class SilentClaimStrategy(
         storeCredentials(entry.key, credentials).getOrNull()?.let {
             accountService.getAccountForWallet(entry.key)?.run {
                 createEvents("", this, entry.value, EventType.Credential.Receive)
-                createNotifications(this, credentials, EventType.Credential.Receive)
+                if (ConfigManager.getConfig<NotificationConfig>().url.startsWith("http")) {
```

Comment:
I understand that notification and trust configs are both part of the silent-exchange feature in the catalog, but I'd still avoid calling them directly, without checking if feature is enabled and generally speaking, I'd avoid their dependency unless explicitly required for the scope of the entity, e.g. [trust-config](https://github.com/walt-id/waltid-identity/blob/0dfa9f0a0f70e53554421e113a69db886f9841d7/waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/service/WalletServiceManager.kt#L81) - although there are 4 entities depending on this value: issuer-/verifier- name resolution and issuer-/verifier- trust validation services, none of them has a direct dependency on the config itself